netiquette/README.md

```diff
@@ -0,0 +1,63 @@
+# BcnEng's slack netiquette
+
+Welcome to BcnEng's slack netiquette. If you don't agree with some guideline or whish to propose an improvement, feel free to open a pull request.
+
+## Definition
+
+From [wikipedia](https://en.wikipedia.org/wiki/Etiquette_in_technology#Netiquette):
+
+> Netiquette [...] is a set of social conventions that facilitate interaction over networks [...]
+
+## Conventions
+
+Here you will find a set of basic conventions that we find useful.
```

Comment:
I would say:

> [...] we find useful. They are part of the set of behaviours that the community users expect from each other. Please make sure you understand all of them. Remember that you can write any question or ask for advice in our specific Slack channel `#coc-netiquette`

Of course we must create that channel :)

Reply:
I just created the channel. But feel free to rename it, remove it, archive it, it it it.